xilem/Cargo.toml

```diff
@@ -91,6 +91,10 @@ path = "examples/emoji_picker.rs"
 # cdylib is required for cargo-apk
 crate-type = ["cdylib"]
 
+[[example]]
+name = "gallery_label_xilem_view"
+path = "examples/gallery_label_xilem_view.rs"
+
```

Comment:
This example doesn't need to be defined in `Cargo.toml`. The others that are defined there are hacks for Android examples, which this one doesn't need.

```suggestion
```